packages/server/controllers/guilds.ts

```diff
@@ -0,0 +1,21 @@
+import prisma from "../utils/prisma";
+import { Guild } from ".prisma/client/index";
+
+async function getAllGuilds() {
+  const query = await prisma.guild.findMany();
+  return query;
+}
```

Comment:
We should write down the return type annotations for all the controller functions for clarity.